src/Generators/NovaTestGenerator.php

```diff
@@ -60,24 +60,23 @@ public function generateTests(): void
         $actions = $this->getActions();
         $filters = $this->collectFilters();
 
-        $resourceClass = Str::afterLast($this->novaResourceClassName, '\\');
-
         $fileContent = $this->getStub('nova_test', [
-            'entity_namespace' => $this->generateNamespace($this->paths['models'], $this->modelSubFolder),
+            'entity_namespace' => $this->generateNamespace('models', $this->modelSubFolder),
             'entity' => $this->model,
-            'resource_name' => $resourceClass,
+            'resource_name' => $this->getTestingEntityName(),
             'resource_namespace' => $this->novaResourceClassName,
-            'snake_resource' => Str::snake($resourceClass),
+            'snake_resource' => Str::snake($this->getTestingEntityName()),
             'dromedary_entity' => Str::lcfirst($this->model),
             'lower_entities' => $this->getPluralName(Str::snake($this->model)),
             'actions' => $actions,
             'filters' => $filters,
-            'models_namespace' => $this->generateNamespace($this->paths['models']),
+            'models_namespace' => $this->generateNamespace('models', $this->modelSubFolder),
+            'user_model_path' => $this->generateNamespace('models') . '\User',
```

Comment:
why are you adding a separate `user_model_path` and use it instead of `models_namespace` and didn't remove the `models_namespace` as it not using anymore?

please revert it as it good solution for now to pass the base models namespace and add the User